mkosi.extra/usr/lib/tmpfiles.d/etc.conf

```diff
@@ -63,5 +63,11 @@ L? /etc/alternatives
 # PackageKit does not run without /etc/PackageKit/ and GNOME stalls
 # logout/reboot if it doesn't run.
 L? /etc/PackageKit
-# ModemManager needds its dbus policy file
-L? /etc/dbus1/systemd.d/org.freedesktop.ModemManager1.conf
+# ModemManager/WPASupplicant/Flatpak need their dbus policy files
+L? /etc/dbus-1/system.d/org.freedesktop.ModemManager1.conf
+L? /etc/dbus-1/system.d/wpa_supplicant.conf
```

Comment:
@keszybz @DaanDeMeyer any chance a PR could be filed to move this to /usr/share please? Ubuntu already installs it there, it's this line: https://src.fedoraproject.org/rpms/wpa_supplicant/blob/rawhide/f/wpa_supplicant.spec#_145